app.js

```diff
@@ -14,12 +14,52 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
-  res.render('index');
+  res.render('index.hbs');
 });
 
-app.listen(3000, () => console.log('🏃‍ on port 3000'));
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi =>
+      //console.log('Beers from the database: ', beersFromApi)
+      res.render('beers.hbs', {
+        beersFromApi
+      }))
```

Comment:
if you are passing a single obj to the render you can put it all in a single line.